app/javascript/components/profile/contributions-summary/use-chart.ts

```diff
@@ -31,30 +31,38 @@ const padReputation = (reputation: number[]): number[] => {
   return reputation.map((r) => min + r)
 }
 
-const CONFIG: ChartConfiguration<'radar'> = {
-  type: 'radar',
-  data: {
-    labels: [],
-    datasets: [],
-  },
-  options: {
-    aspectRatio: 1,
-    elements: {
-      line: { borderWidth: 3 },
+const generateConfig = (
+  canvas: HTMLCanvasElement
+): ChartConfiguration<'radar'> => {
+  const borderColor = getComputedStyle(canvas).getPropertyValue(
+    '--chartBorderColor'
+  )
```

Comment:
@kntsoriano Check the commit I added that changed this to make sure you're happy with it pls.

Reply:
Looks good.